src/plugins/intel_cpu/src/config.h

```diff
@@ -61,10 +62,20 @@ struct Config {
 
     std::string execGraphPath;
     std::string verbose;
-    std::string blobDumpDir = "mkldnn_dump";
+    std::string blobDumpDir = ".cpu_dump";
```

Comment:
renamed mkldnn_dump to cpu_dump
also added dot as a prefix, to make folder hidden for 'ls'